src/SFA.DAS.EmployerIncentives.Commands/EarningsResilienceCheck/EarningsResilienceApplicationsCheckCommandHandler.cs

```diff
@@ -29,6 +33,12 @@ public async Task Handle(EarningsResilienceApplicationsCheckCommand command, Can
                 {
                     tasks.Add(_domainEventDispatcher.Send(domainEvent, cancellationToken));
                 }
+                foreach (var apprenticeship in application.Apprenticeships)
```

Comment:
Should this only apply to incentives where PendingPayments.Count > 0 ?
i.e. should it be a separate resilience check (separate command handler) for that scenario since it's a different use case? 

So look up incentives with pending payments and for those that don't have apprentices with EarningsCalculated true then fire this event?

Reply:
I think that would have to be in a separate handler, the earnings resilience handler works with applications, not the incentives, so would be across different bounded contexts